windows/jitsi-meet/index.html

```diff
@@ -18,6 +18,7 @@
         </style>
     </head>
     <body>
+      <script src="https://webrtc.github.io/adapter/adapter-latest.js"></script>
```

Comment:
We cannot do this. Things may break in different ways. If the Chrome version in Electron doesn't have RTCPeerConnection unprefixed, use webkitRTCPeerConnection please.